firmware/src/rdm_responder.c

```diff
@@ -392,6 +398,9 @@ int RDMResponder_GetSupportedParameters(const RDMHeader *header,
       case PID_SOFTWARE_VERSION_LABEL:
       case PID_DMX_START_ADDRESS:
       case PID_IDENTIFY_DEVICE:
+        if (g_responder->is_subdevice) {
```

Comment:
I think we need to open a bug in E1.20 too then, section 10.4.1 says "PIDs that are included in the minimum support list, indicated by the “Required” column of Table A-3, shall not be reported.". Incidentally "This document does not change or modify the requirements stated in ANSI E1.20. Requirements stated in this document are in addition to the stated ANSI E1.20 requirements." that from E1.37-1 seems a bit contradictory, there are some requirements from E1.20 which adding to would involve changing, e.g. do/don't report required parameters.